src/app/features/tech-record/tech-record.module.ts

```diff
@@ -14,6 +14,9 @@ import { ReactiveFormsModule } from '@angular/forms';
 import { TyresSearchComponent } from './components/tyres-search/tyres-search.component';
 import { TechRecordChangeStatusComponent } from './components/tech-record-change-status/tech-record-change-status.component';
 import { TechRecordTitleComponent } from './components/tech-record-title/tech-record-title.component';
+
+import { ChangeVehicleTypeComponent } from './components/change-vehicle-type/change-vehicle-type.component';
+
```

Comment:
Why the empty lines?
```suggestion
import { ChangeVehicleTypeComponent } from './components/change-vehicle-type/change-vehicle-type.component';
```

Reply:
Not sure but they're not there anymore